### ISSUE TITLE
autoReconnect=true

### ISSUE THREAD

**Issue by Kervinou** [marcelo-mason/PreciousStones#1026]
Hi !

I have a test server that is few used

If i connect after a long time, this console ERROR message appears
It seems that it is in source code that there's something to do

what do you think of that ?

thank you

```
[19:47:38 ERROR]: Error at SQL Query: The last packet successfully received from the server was 56827293 milliseconds ago.  The last packet sent successfully to the server was 56827293 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.
[19:47:38 ERROR]: Query: SELECT * FROM pstone_players WHERE uuid = '2e148fbf-e82b-40b7-b586-0b96bb8fee38';

```

**Comment by frizzbee30** [comment]
Hi, we have had this for quite some time, I think it is just a timeout?

**Comment by Kervinou** [comment]
Yeah it is a timeout
the option autoReconnect=true seems to be the solution
but where put it ? i read that it is when client access mysql so in our case this is in the source code

**Comment by Kervinou** [comment]
Thank you !!!

 

 

De : Phaed [mailto:notifications@github.com] 
Envoyé : mardi 22 novembre 2016 21:04
À : marcelo-mason/PreciousStones
Cc : Kervinou; Author
Objet : Re: [marcelo-mason/PreciousStones] autoReconnect=true (#1026)

 

Closed #1026 <https://github.com/marcelo-mason/PreciousStones/issues/1026>  via  <https://github.com/marcelo-mason/PreciousStones/commit/bcbea142649bc3ca1ab72698e7b9fac181243f3b> bcbea14.
